libc/include/llvm-libc-macros/fenv-macros.h

```diff
@@ -9,11 +9,12 @@
 #ifndef LLVM_LIBC_MACROS_FENV_MACROS_H
 #define LLVM_LIBC_MACROS_FENV_MACROS_H
 
-#define FE_DIVBYZERO 1
-#define FE_INEXACT 2
-#define FE_INVALID 4
-#define FE_OVERFLOW 8
-#define FE_UNDERFLOW 16
+#define FE_DIVBYZERO 0x1
+#define FE_INEXACT 0x2
+#define FE_INVALID 0x4
+#define FE_OVERFLOW 0x8
+#define FE_UNDERFLOW 0x10
+#define __FE_DENORM 0x20
```

Comment:
Shall we all so define it in `hdr/fenv_macros.h` for overlay mode if the system header does not define it?  For completeness and hermeticity?

Reply:
Hm, unsure because if it comes from the system we don't necessarily know what value to assign to it. I think GNU uses 0x2 or something.

Reply:
"I am the system" -libc

Reply:
I mean in overlay mode we just rely on the GNU headers in practice, it seems non-trivial to select a value that's not going to conflict, unless 1 << 31 or something.

Reply:
For overlay mode, anything we fill in `hdr/fenv_macros.h` should only be used internally.  So it is kind of the best place to filling in any macros that are missing from the system/compiler headers.  And if it's not presented in the system header, it shouldn't matter much which value you assign to it, as long as it is different from the rest of the `FE_*` right?

Reply:
Alright, I'll just set it to `1 << 31` then, since it's supposed to be a power of two and that's highly unlikely to be taken.

Reply:
Would be better to keep it as small as possible, since all the powers of 2 up to 1 << 6 are free constants

Reply:
maybe something like (`MAX(FE_DIVBYZERO, FE_INEXACT, FE_INVALID, FE_OVERFLOW, FE_UNDERFLOW) << 1)` if it's not too much work :D.

Reply:
Hm, for now `64` is probably safe enough. Unsure if it's worth the effort since we'd need to make some variadic max function. That'd probably be doable with constexpr functions and C++ magic, but with macros it seems like a big pain unless we want to invoke an actual function call.

Reply:
For AMDGPU the best values to use would just be to directly take the bit positions from TRAP_STS. i.e. INPUT_DENORMAL is bit 1, FE_INVALID is 0, and so on

Reply:
That's probably occupied by something else, internally we map from the "real" values to the "libc" ones.

Reply:
The standard is implementation defined power of 2 for all of these. We can just define them to what we want